### ISSUE TITLE
Intiallizing the weight matrix

### ISSUE THREAD

**Issue by vanrajbrown** [MuhammedBuyukkinaci/TensorFlow-Multiclass-Image-Classification-using-CNN-s#3]
How are you initializing weight matrix in multiclass_classification_gpu.py line number 83. Although when avoiding vanishing and exploding gradient, we need to use np.random.randn(**size_l, size_l-1**). Can you please explain the numbers you are using - tf.truncated_normal([11,11,3,96], stddev=0.01).
Thanks

**Comment by vanrajbrown** [comment]
Okay, I got the reason as to why are you using these number, these are from AlexNet Architecture. But do you have any write up written on this, as to why are you using what you are on each step. That would be so helpful, I have lot more questions, will it be fine if I post them under the issues page?

**Comment by MuhammedBuyukkinaci** [comment]
Sorry for being late. I tried to implement AlexNet Paper. The hyperparameters used in the model are from that paper. You are highly welcome to ask your questions here. I will try to answer them.

**Comment by vanrajbrown** [comment]
Thanks, I Figured out the Alex net Architecture. I have another question, why the value of Bias constant varies either 0 or 1. How do you decide which layer bias value as 1 or 0 ?

**Comment by MuhammedBuyukkinaci** [comment]
I just got this information from CS224 of Andrej Karpathy on youtube.com .
